api_fhir_r4/converters/medicationConverter.py

```diff
@@ -170,14 +172,16 @@ def build_fhir_medication_type_extension(cls, value):
 
     @classmethod
     def build_fhir_medication_frequency(cls, fhir_medication, imis_medication):
-        medication_frequency = cls.build_fhir_medication_frequency_extension(imis_medication.frequency)
-        if type(fhir_medication.extension) is not list:
-            fhir_medication.extension = [medication_frequency]
-        else:
-            fhir_medication.extension.append(medication_frequency)
+        if imis_medication.frequency:
+            medication_frequency = cls.build_fhir_medication_frequency_extension(imis_medication.frequency)
+            if type(fhir_medication.extension) is not list:
+                fhir_medication.extension = [medication_frequency]
+            else:
+                fhir_medication.extension.append(medication_frequency)
 
     @classmethod
     def build_fhir_medication_frequency_extension(cls, value):
+        # TODO: Is this ok? Value is assigned to period instead of frequency
```

Comment:
This follows the IG: https://fhir.openimis.org/StructureDefinition-openimis-medication-definitions.html#Medication.extension:timing